### ISSUE TITLE
False Positive: bbci.co.uk

### ISSUE THREAD

**Issue by DRSDavidSoft** [jawz101/MobileAdTrackers#21]
Firstly, I'd like to appreciate your efforts in collecting and maintaining this useful list!

The domain name `bbci.co.uk` is used to load images from the BBC website. It appears that it was added around three months ago via f010fde.

You can confirm that the domain name is not exclusively used to load ads, by visiting the https://www.bbc.com/ homepage. As an example, here's the first link that appears on their homepage as of writing this:

- https://www.bbc.com/news/science-environment-48966645

Notice where the main article's image is being retrieved from:

```
https://ichef.bbci.co.uk/news/320/cpsprodpb/18CA/production/_107864360_899a5d13-e2da-47f7-8fe9-bd29670246bb.jpg
```

However, if the `bbci.co.uk` domain name is blocked, none of the article images will be loaded.

So, I'd like you to whitelist the `bbci.co.uk` domain. Thank you! :)

**Comment by jawz101** [comment]
removed.  Thanks.  I don't know how that got in there

**Comment by DRSDavidSoft** [comment]
@jawz101 Sure thing, thanks for creating and maintaining this awesome list!

---

<small>P. S. a tip: if you write "resolves #21" as your commit message, it'll automatically close this issue and marks it as solved. (21 being the issue's number)</small>

Cheers!
